parallaxinc/solo
# Solo 192 refactor project empty detection

Refactors empty code detection on initialization

To test:
Open a project, make sure it appears.
There isn't a clear way to test loading an empty project...
